### ISSUE TITLE
error in solvers.py: assert glpk.glp_get_col_kind(prob, j) == glpk.GLP_IV 

### ISSUE THREAD

**Issue by pchtsp** [coin-or/pulp#72]
For some reason this line is givng me troubles when trying to use the PYGLPK() class. I commented it out and it "works" (not sure if incorrectly though).

It is the only assertion in the file: is there a particular reason this is here? Why could it be complaining?

(I installed pulp via easy_install -U pulp).

thanks!

**Comment by stumitchell** [comment]
How did you install pyGLPK?

I tried to add it to travis testing but couldn't find an easy way.

Stu

**Comment by pchtsp** [comment]
It is possible that I did not have it installed. I downloaded the source
and attempted to compile but it failed several times during the
compilation.

I have the python-glpk bindings that I installed with sudo apt-get install
in Ubuntu.

What is weird is that it "works" until the assertion. Since it worked I
assumed I managed to install it somehow.

How can I know if I have it correctly installed?

Thanks for the help!

Sent from my netBook

How did you install pyGLPK?

I tried to add it to travis testing but couldn't find an easy way.

Stu

--
Reply to this email directly or view it on GitHub
<https://github.com/coin-or/pulp-or/issues/72#issuecomment-89147021>.
